buildconfig/manylinux-build/build-wheels.sh

```diff
@@ -1,17 +1,12 @@
 #!/bin/bash
 set -e -x
 
+# This file is not used by CI anymore. This only exists incase someone wants to
+# generate manylinux wheels locally for testing
```

Comment:
What's the point of all the changes to this file if it's not being used anymore in CI? Would anyone actually use this locally? Have you tested these changes locally?

Reply:
Can't speak for others, but I do use this file locally, and often with a lot of custom changes ;)
I've also tested these changes